python/hail/methods/family_methods.py

```diff
@@ -55,3 +56,172 @@ def trio_matrix(dataset, pedigree, complete_trios=False):
     """
     return MatrixTable(dataset._jvds.trioMatrix(pedigree._jrep, complete_trios)
                        .annotateGenotypesExpr('g = {proband_entry: g.proband, father_entry: g.father, mother_entry: g.mother}'))
+
+@handle_py4j
+@require_biallelic
+@typecheck(dataset=MatrixTable,
+           pedigree=Pedigree)
+def mendel_errors(dataset, pedigree):
+    """Find Mendel errors; count per variant, individual and nuclear family.
+
+    .. include:: ../_templates/req_tvariant.rst
+    .. include:: ../_templates/req_tstring.rst
+    .. include:: ../_templates/req_biallelic.rst
+
+    Examples
+    --------
+
+    Find all violations of Mendelian inheritance in each (dad, mom, kid) trio
+    in a pedigree and return four tables:
```

Comment:
Since this is before the description of what the tables are, could you maybe put a small description of them? like `(all errors, errors by family, errors by individual, errors by variant)` or something?